content/providers/03-community-providers/11-workers-ai.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: Workers AI
+description: Learn how to use the Workers AI provider with the Vercel AI SDK.
+---
+
+# Workers AI
+
+[threepointone/workers-ai-provider](https://github.com/threepointone/workers-ai-provider/) is a community provider that enables Cloudflare's [Workers AI](https://ai.cloudflare.com/) models for use the Vercel AI SDK.
+
+## Setup
+
+The Workers AI provider is available in the `workers-ai-provider` module. You can install it with:
+
+<Tabs items={['pnpm', 'npm', 'yarn']}>
+  <Tab>
+    <Snippet text="pnpm add workers-ai-provider" />
+  </Tab>
+  <Tab>
+    <Snippet text="npm install workers-ai-provider" />
+  </Tab>
+  <Tab>
+    <Snippet text="yarn add workers-ai-provider" />
+  </Tab>
+</Tabs>
+
+### Enabling AI in Workers Project
+
+First, setup an AI binding in your Workers project `wrangler.toml` file:
+
+```bash filename="wrangler.toml"
+# ...
+[ai]
+binding = "AI"
+# ...
+```
+
+### Usage
```

Comment:
It would be good if we could follow the same section structure as in the OpenAI provider everywhere. Standardization helps a lot with automation later, and users have an easier time orienting themselves.

Reply:
updated